src/app/products/share-products/dividends-share-product/dividends.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+    <button mat-raised-button color="primary">
+      <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+      Create Dividend
+    </button>
+</div>
+<div class="container">
+
+  <div fxLayout="row">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+
+  <div class="mat-elevation-z8">
+
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+        <td mat-cell *matCellDef="let dividend">{{ dividend.productData.name }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="dividendPeriodStartDate">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Dividend Period Start Date </th>
+        <td mat-cell *matCellDef="let dividend">{{ dividend.dividendPeriodStartDate | date }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="dividendPeriodEndDate">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Dividend Period End Date </th>
+        <td mat-cell *matCellDef="let dividend">{{ dividend.dividendPeriodEndDate | date }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="amount">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Amount </th>
+        <td mat-cell *matCellDef="let dividend"> {{ dividend.amount }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="status">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let dividend">{{ dividend.status }}</td>
```

Comment:
```suggestion
        <td mat-cell *matCellDef="let dividend">{{ dividend.status.value }}</td>
```
I guess you couldn't find a working test case? change url to `demo.mifos.io` share product `8` has a list of dividends.

Reply:
It shows invalid user details when I login.

Reply:
Yes, there were some changes made to auth recently, login using dev.mifos.io check the remember me checkbox, once you are logged in change it to demo.mifos.io.